io.github.gitahead.Gitahead.json

```diff
@@ -0,0 +1,123 @@
+{
+  "app-id": "io.github.gitahead.Gitahead",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15",
+  "sdk": "org.kde.Sdk",
+  "command": "/app/usr/local/GitAhead",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=dri",
+    /*for git repos that require ssh keys*/
+    "--socket=ssh-auth",
+    "--share=network",
+    "--socket=pcsc",
+    /*we want access to all files under home*/
+    "--filesystem=home",
+    /*we use the keyring to store credentials*/
+    "--filesystem=xdg-run/keyring",
+    "--talk-name=org.freedesktop.secrets",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.freedesktop.Flatpak"
+  ],
+  "modules": [
+    "shared-modules/libsecret/libsecret.json",
+    {
+      "name": "libsectre"
+    },
+    "shared-modules/intltool/intltool-0.51.json",
+    {
+      "name": "intltool"
+    },
+    {
+      "name": "git",
```

Comment:
It's likely a better idea to build it from source than hope that copying binaries from Sdk will always work.

Reply:
okay, I'll change it to build from source